app/components/schools/teacher_profile_summary_list_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<%=
+  govuk_summary_list(
+    rows:
+  )
+%>
```

Comment:
If this is all the component does you don't really need a template, just do

```ruby
def call
  govuk_summary_list(rows:)
end
```

Or including the header

```ruby
def call
  safe_join([tag.h2('hi'), govuk_summary_list(rows:)])
end
```

Reply:
Thank you, this is awesome! 😍 

https://github.com/DFE-Digital/register-early-career-teachers-public/pull/313/commits/b046bd3da49319c6f229e0dfb9802e10d10a7f78